pyproject.toml

```diff
@@ -31,7 +31,9 @@ dependencies = [
     "termcolor",
     "tomli",
     "pasha",
-    "extra_geom"
+    "extra_geom",
+    "zulip",
+    "tabulate"
```

Comment:
```suggestion
```

Reply:
Tabulate must be [installed](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.to_markdown.html) to use `to_markdown`. But you are right, I don't need to import it.